core/serializers.py

```diff
@@ -55,6 +57,14 @@ def to_representation(self, instance):
                 ]
             }
 
+    def estimate_count(self):
+        if self.operation:
+            MAX_OFFSET = 65390
+            count, data = query.query_table(
+                self.operation, 1, MAX_OFFSET, estimate_count=True)
+            return len(data) > 0
+        return True
```

Comment:
I found this to be wildly inaccurate ... which is why I added the asynchronous `row_count` feature that returns an accurate count via celery ... actually, this looks like something I wrote a long time ago that I have since removed ... I worry you've got a bit of old code.

Reply:
Removed